SP-4105 (#28)
stackpulse/playbooks: README.md

<br>

<table border=2 border-style=outset>
  <tr>
    <td border=0 width="110">
      <a href="https://github.com/stackpulse/playbooks/tree/master/stackpulse/create-incident-war-room">
      <img src="images/stackpulse.png"/>
      </a>
    </td>
    <td>

>       <a href="https://github.com/stackpulse/playbooks/tree/master/stackpulse/create-incident-war-room"><b>Create Incident War-room (Slack)</b></a><br><br>Playbook that creates Incident War Room in Slack and invites the relevant participants based on incident details.<br><br>

Reviewer: ```suggestion
      <a href="https://github.com/stackpulse/playbooks/tree/master/stackpulse/create-incident-war-room"><b>Create Incident War-room (Slack)</b></a><br><br>This playbook creates a Slack Incident War Room and invites participants to it.<br><br>
```
